components/net/bluetooth_thread.rs

```diff
@@ -48,6 +49,19 @@ macro_rules! return_if_cached(
     );
 );
 
+macro_rules! return_if_blacklisted(
+    ($blacklist_result:expr, $sender:expr, $exclude_fn:ident, $uuid:expr) => (
+       if let Err(ref err) = $blacklist_result {
+            return drop($sender.send(Err(err.clone())));
+        } else {
```

Comment:
we need this else branch beacause if we leave this, the borrow checker throws an error